pages/contact/index.tsx

```diff
@@ -7,7 +7,7 @@ const index = () => {
         <h1>Contact</h1>
         <div className={styles.hero}>
           <div className={styles.container}>
-            <h1>Email Us: *insert email here*</h1>
+            <h1>Email Us: <a href='mailto:20dollarinsuranceproduction@gmail.com'>20DollarInsuranceProduction@gmail.com</a></h1>
```

Comment:
Didn't realize the email address was that bulky. How about changing to just an `h1` tag saying `Email Us` that links to our email.

Reply:
fixed in the latest push: fix pr comment to put only email us on contact page